auto_tests/group_message_test.c

```diff
@@ -100,10 +100,10 @@ static void group_message_handler(Tox *tox, uint32_t groupnumber, uint32_t peer_
     self_name[self_name_len] = 0;
 
     ck_assert(s_err == TOX_ERR_GROUP_SELF_QUERY_OK);
-    ck_assert(memcmp(self_name, PEER0_NICK, self_name_len));
+    ck_assert(!memcmp(self_name, PEER1_NICK, self_name_len));
 
     printf("%s sent message to %s: %s\n", peer_name, self_name, message_buf);
-    ck_assert(memcmp(message_buf, TEST_MESSAGE, length) == 0);
+    ck_assert(!memcmp(message_buf, TEST_MESSAGE, length));
```

Comment:
Prefer `== 0`, because `int` is not an Essentially Boolean type (MISRA C:2012 8.10.2). I know `!memcmp` and `!strcmp` is a common idiom in C, but prefer strong typing over poor (even if common) idioms.